### ISSUE TITLE
Define and validate the v1 browser candidate-bundle manifest

### ISSUE THREAD

**Issue by peterbucci** [peterbucci/signlab#34]
## Parent epic

- #5

## Outcome

Define the smallest immutable browser **candidate-bundle** envelope needed for #35 to package the exact checkpoint nominated by #32.

This story defines and validates structure, identity, and compatibility. It does not export, execute, publish, promote, activate, or load a model.

## Why this story was narrowed

The original wording duplicated existing feature, taxonomy, segmenter, calibration, and policy artifacts, while also pulling in exporter, atomic publication, parity, and browser-loader work already owned by #35, #36, #37, #39, and #40.

SignLab already has authoritative v1 documents for landmark extraction, quality, the 64×126 hand-local feature plan, candidate-event segmentation, and the constructed decision policy. The bundle must reference and reconcile those exact documents, not create parallel labels/input/preprocessing/calibration/threshold schema families.

There is no promoted model or champion. #32 nominated one development checkpoint for portable export/evaluation only.

## Planning

- **Phase:** Phase 3 — Portable inference
- **Priority:** P0
- **Effort:** M

## Acceptance criteria

- [x] Define one strict `browser-model-bundle/1` Python manifest and one generated standalone Draft 2020-12 JSON Schema, separate from retained research `model-manifest/1`.
- [x] Bind candidate-only status, bundle ID/version, nomination/code/run/data/corpus/split/taxonomy/checkpoint identities, fixed ONNX tensor contract/opset, label order, landmarker compatibility, component identities, licenses, and a sorted bundle-relative asset inventory.
- [x] Reuse the exact v1 landmarker, quality, feature, segmenter, and decision-policy documents; add only the missing strict v1 decision-policy reader. Do not define duplicate labels, input, preprocessing, calibration, or threshold schemas.
- [x] Require checksum-bound roles for `model.onnx`, landmarker config, quality policy, feature plan, segmenter, decision policy, model card, and at least one golden JSON smoke asset.
- [x] Bind both exact MediaPipe hand/pose task hashes and sizes as runtime requirements without copying the 13.6 MB of shared task bytes into every model bundle.
- [x] Define the candidate input as `[1,64,126]` float32 and output as `[1,6]` float32 class probabilities in the exact six-label order; do not call the current output logits.
- [x] Give the canonical manifest a domain-separated digest; because it contains every asset hash, that digest transitively identifies the complete bundle.
- [x] Validate canonical manifest bytes and fail closed on unsupported versions, unsafe paths, symlinks, missing/extra assets, byte-count/hash mismatch, duplicate roles/paths, or cross-component identity/shape/label mismatch.
- [x] Publish one canonical synthetic example manifest and tests that materialize harmless structural bytes; do not commit a fake ONNX graph or claim runtime validity.
- [x] Document v1 as exact and immutable: unknown versions and silent migration are rejected; changed assets create a new digest, while changed schema semantics/required roles require `/2`.
- [x] Represent unresolved trained-model distribution terms honestly as `NOASSERTION` / local evaluation only until a public-distribution decision is reviewed.

## Required bundle roles

```text
manifest.json
model.onnx
landmarker.json
quality-policy.json
feature-plan.json
segmenter.json
decision-policy.json
model-card.md
golden/smoke.json
```

The manifest is the checksum inventory; no redundant `checksums.json` is needed. Labels and tensor contracts are inline. Calibration and abstention remain in the decision policy. Mirroring, handedness, feature order, interpolation, padding, and masks remain in the feature/quality contracts.

## Evidence of completion

The future #35 exporter can import the Python manifest/validator, and the future TypeScript loader can consume the same JSON Schema and example. A byte, path, version, role, tensor, label, or component-identity mutation fails with a stable path-safe reason.

## Hard stop

Maximum **550 physical lines** across production Python, tests, and any schema-writer script. One production module, one test module, one generated schema, one example, one short documentation update, and no new dependency.

Stop and re-scope if work starts adding a registry, general storage abstraction, schema family, migration engine, signing system, DVC integration, publication service, or release-state machine.

## Explicit deferrals

- Keras-to-ONNX conversion, ONNX checker/CPU smoke, real golden generation, deterministic bytes, and atomic content-addressed publication: #35.
- TypeScript preprocessing/quality/decision behavior: #36.
- Native/Keras-to-ONNX numerical parity: #37.
- Full Python/ONNX/TypeScript/ONNX Web goldens: #39.
- Fetching, checksum validation in TypeScript, caching, activation, rollback, offline behavior, and network recovery: #40.
- Browser segmentation/inference/replay and end-to-end performance: #43, #44, #45, and #51.
- Champion or release decisions and natural-use claims: later release evidence; #18 remains optional natural-session work.

## Dependencies

- #22 — portable landmark representations
- #29 — candidate-event detector
- #31 — decision policy/calibration mechanics
- #32 — exact nominated development candidate

### Completion record

- Merged in PR #101 at commit `48eee3d0aa324504cf7258c6bde39ac1f024ce50`.
- Scope guard: 335 production + 214 test lines = **549/550**.
- Focused verification: 24 passed; the locally unavailable symlink case passed in Linux CI.
- Full web, secret-scan, Linux quality/test/package, and Windows locked-install/test/package jobs passed.
- Two independent final reviews approved correctness and scope.
